client/client.go

```diff
@@ -393,3 +393,14 @@ func (client *Client) CreateUser(request *CreateUserRequest) (*CreateUserResult,
 
 	return &createResult, nil
 }
+
+// UpdateRevisions asks the daemon to sync the core/kernel snap revisions
+// with what was booted
+func (client *Client) UpdateRevisions() error {
+	var rsp interface{}
+	if _, err := client.doSync("POST", "/v2/update-revisions", nil, nil, nil, &rsp); err != nil {
```

Comment:
Yeah, I assumed we were going for the latter option, with the underlying "boot-ok" logic also being done by the daemon, unprompted. The motivation for this would be the same as the one for taking the old "firstboot" logic out.

If we need to block the boot process until a given point is reached, I think it'd be fine to have an external way to ask for that, but it would be informational rather than starting a fundamental action from the outside. If that's needed, please let me know and we can quickly design something for it.